spec/mailers/previews/user_mailer_preview_spec.rb

```diff
@@ -13,7 +13,9 @@
   it 'has a preview method for each mailer method' do
     mailer_methods = UserMailer.instance_methods(false)
     preview_methods = UserMailerPreview.instance_methods(false)
-    expect(mailer_methods - preview_methods).to be_empty
+    expect((mailer_methods - preview_methods).sort).to eq(
+      [:email_address, :user, :validate_user_and_email_address].sort,
+    )
```

Comment:
i would just do another subtraction:
```suggestion
    mailer_helper_methods = [:email_address, :user, :validate_user_and_email_address]
    expect(mailer_methods - mailer_helper_methods - preview_methods ).be_empty
```